### PR TITLE
moyo-header project is done

### PR DESCRIPTION
**Pull request by OksanaK24**
[DEMO LINK](https://oksanak24.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://oksanak24.github.io/layout_moyo-header/report/html_report/)

- [x] Header height is set in 1 place (for the links)
- [X]  Content is vertically centered (for any header height)
- [X]  CSS is used to show all letters in Uppercase (don't type them in HTML)
- [X]  Logo is an image wrapped with a link
- [X]  CSS Variable is used for a blue color
- [X]  Pseudo-element is used for a blue line below the active link
- [X]  Code follows all the [Code Style Rules ❗️](https://github.com/OksanaK24/layout_moyo-header/blob/master/checklist.md)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,9 +6,46 @@
...
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html">
+        <img src="./images/logo.png" alt="logo"
+          class="logo-img">
```
Comment (etojeDenys) [inline src/index.html:19]: Pasted Graphic

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,84 @@
+html {
+  margin: 0;
+  padding: 0;
+
+  --active-color: #00ACDC;
+  --text-color: #000;
+  --background: #FFF;
```
Comment (etojeDenys) [inline src/style.css:7]: move to ::root selector all vars

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,84 @@
...
+  margin-left: 0;
+}
+
+.menu-item a:hover {
+  color: var(--active-color);
+}
+
+.is-active {
+  color: var(--active-color);
+}
```
Comment (etojeDenys) [inline src/style.css:72]:
```suggestion
.menu-item a:hover,
.is-active {
  color: var(--active-color);
}
```

**Review by etojeDenys — CHANGES_REQUESTED** [review]
try to fix your demo links

**Comment by OksanaK24** [comment]
I would need help with deployment... I have no idea why it doesn't work as it is supposed to =(

**Pull request closed without merging**